src/commands/violations/create.js

```diff
@@ -51,7 +50,7 @@ class CreateViolation extends Command {
 			.setAuthor("FAGC Community")
 			.setDescription(`Create FAGC violation for \`${playername}\``)
 		embed.addFields(
-			{ name: "Admin name", value: admin_name, inline: true },
+			{ name: "Admin user", value: `<@${message.author.id}> | ${message.author.tag}`, inline: true },
```

Comment:
Why do you include both id and tag?
This would produce `@Cooldude2606 | Cooldude2606#2606` which seams very redudent.

Reply:
MessageEmbeds aren't supposed to include mentions, so the user doesn't have to be cached. It therefore just shows a `<@userID>`, such as `<@429696038266208258>`

Reply:
Resolved on Discord